app/styles/landing.css

```diff
@@ -0,0 +1,62 @@
+.landing-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-content: center;
+  height: 80vh;
+  width: 70%;
+  margin: auto;
+}
+
+.landing-container h2, h3 {
+  color: black;
+}
+
+.landing-container h2 {
+  font-size: 32px;
```

Comment:
@sumitd94 , any particular reason for going with px and not rem?

Reply:
No specific reason .. not used to using `rem` 😅  should I change it to `rem`?

Reply:
@sumitd94 Yes, if some user changes the default font size for their browser to say 24px, since you've hardcoded the font-size here, your text would remain fixed irrespective of the changes done by the user. 
Better to use `rem` for font-size 👍🏻

Reply:
Sure, will change this.

Reply:
e823a79